### ISSUE TITLE
Does not support globs

### ISSUE THREAD

**Issue by maxlapides** [bitrise-steplib/bitrise-step-flutter-test#3]
Consider this configuration:

```yaml
- flutter-test@0.9.1:
        inputs:
        - additional_params: "lib/**/*_test.dart"
```

In this case, it will run

```bash
flutter "test" "lib/**/*_test.dart" 
```

Because the glob pattern is wrapped in quotes, the glob will not be expanded.

**Comment by maxlapides** [comment]
The recommended solution is to move all the test files into a `test` directory at the project root. However, this is still an open issue.

**Comment by mefiblogger** [comment]
Hello,

It's Mefi here from the Bitrise CoreSteps Team. Thanks for reaching out helping us and the mobile community to have a better experience with Bitrise steps. 💜

We started to have a broader review of all the open issues and PRs and we are continuously prioritizing these in our sprints.

We reviewed this issue and we found it to be a valid bug. We added this ticket to our internal backlog and we'll get back to you once we released the new version with the fix.

Happy building! 🚀
